packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "0.96.1",
+  "version": "0.96.1-fb-misc-ext-dev-cleanup.12",
```

Comment:
don't forget to update the release notes

Reply:
Done